### PR TITLE
Add pigeons

### PR DESCRIPTION
**Pull request by 8street**
#### Summary

SUMMARY: Content "Add pigeons"

#### Purpose of change

The game's cities look a little empty, except the zombies. 

#### Describe the solution

Add some pigeons. Their spawn is the same as the crows spawn in cities. In the wild, pigeons are much less common.

#### Describe alternatives you've considered

Unfortunately, pigeons and crows do not spawn very often in cities, it may be worth creating a new animal spawn group specifically for cities.

#### Testing

Open game, spawn some pigeons.
Wander around and see some more.

### CODE REVIEW

File: data/json/monstergroups/misc.json [hunk 1]
```diff
@@ -115,6 +116,12 @@
     "monsters": [
       { "monster": "mon_fly", "freq": 100, "cost_multiplier": 1 },
       { "monster": "mon_crow", "freq": 100, "cost_multiplier": 1 },
+      {
+        "monster": "mon_pigeon",
+        "freq": 100,
+        "cost_multiplier": 1,
+        "conditions": [ "DAWN", "DUSK", "DAY", "SPRING", "SUMMER", "AUTUMN" ]
```
Comment (Fris0uman) [inline data/json/monstergroups/misc.json:123]: No pigeon in winter? Where do they go? ^^

Comment (8street) [reply data/json/monstergroups/misc.json:123]: As I understand it, the game takes place in mid-latitudes. Then the birds migrate south. Some small birds, such as sparrows or tomtits, never migrate. But pigeons are not one of them.

**Comment by curstwist** [comment]
There is a roof nested mapgen for a chicken coop that should've been for pigeons.  If you'd like to update those spawns.

https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/json/mapgen/nested/nested_chunks_roof.json#L115

this will put them on a few roofs around towns to help your spawn issues.

File: data/json/monstergroups/misc.json (hunk 1 above)
Comment (Fris0uman) [reply data/json/monstergroups/misc.json:123]: I don't think pigeon migrate during winter.

Comment (8street) [reply data/json/monstergroups/misc.json:123]: I'd added winter spawn. See ok for you?

**Comment by 8street** [comment]
Can be merged, I guess.

File: data/json/monstergroups/wilderness.json [hunk 2]
```diff
@@ -2848,6 +2869,20 @@
...
+        "cost_multiplier": 0,
+        "pack_size": [ 1, 7 ],
+        "conditions": [ "DAWN", "DUSK", "DAY", "SPRING", "SUMMER", "AUTUMN" ]
+      },
+      {
+        "monster": "mon_pigeon",
+        "freq": 15,
+        "cost_multiplier": 0,
+        "pack_size": [ 1, 7 ],
+        "conditions": [ "DAWN", "DUSK", "DAY" ]
```
Comment (Fris0uman) [inline data/json/monstergroups/wilderness.json:2884]:
```suggestion
        "conditions": [ "DAWN", "DUSK", "DAY", "WINTER" ]
```

**Pull request merged**